source/includes/kubernetes/_k8_persistentvolumes.md

```diff
@@ -238,6 +238,90 @@ Return value:
 | `taskStatus` <br/>_string_ | The status of the operation.                                     |
 
+<!-------------------- REPLACE PERSISTENT VOLUME -------------------->
+
+#### Replace a persistent volume
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/persistentvolumes/my-persistent-volume"
+  Content-Type: application/json
+  {
+    "apiVersion": "v1",
+    "kind": "PersistentVolume",
+    "metadata": {
+        "annotations": {
+            "key": "value"
+        },
+        "name": "persistentvolumes-name"
+    },
+    "spec": {
+        "accessModes": [
+            "ReadWriteMany"
+        ],
+        "capacity": {
+            "storage": "2Gi"
+        },
+        "claimRef": {
+            "apiVersion": "v1",
+            "kind": "PersistentVolumeClaim",
+            "name": "pv-claim-name",
+            "namespace": "default",
+            "resourceVersion": "259087",
+            "uid": "11957feb-1acc-4c7e-911e-f77d5d7be5ac"
+        },
+        "hostPath": {
+            "path": "/etc/path"
+        },
+        "persistentVolumeReclaimPolicy": "Retain",
+        "storageClassName": "standard",
+        "volumeMode": "Filesystem"
+    }
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumes/:id</code>
+
+Replace a persistent volume in a given [environment](#administration-environments).
+
+Required Attributes                 | &nbsp;
+----------------------------------- | ------------------------------------------------------------
+`apiVersion` <br/>_string_          | The api version (versioned schema) of the volume.
+`metadata` <br/>_object_            | The metadata of the volume.
+`metadata.name` <br/>_string_       | The name of the volume.
+`spec`<br/>_object_                 | The specification used to replace and run the volume.
+`spec.accessModes`<br/>_string_ | The desired access modes the volume should have.
+`spec.capacity.storage`<br/>_string_  | The size of the claim.
+`spec.hostPath.path`<br/>_string_  | The path of the directory on the host.
+
+| Optional Attributes                       | &nbsp;                                                                  |
+| ----------------------------------------- | ----------------------------------------------------------------------- |
+| `metadata.annotations` <br/>_string_        | Annotations is an unstructured key value map stored with a resource that may be set by external tools to store and retrieve arbitrary metadata.|
+| `spec.claimRef` <br/>_string_        | part of a bi-directional binding between PersistentVolume and PersistentVolumeClaim. Expected to be non-nil when bound.|
```

Comment:
All the other attributes begin with a capital letter except for this one. 

`part ...` -> `Part ....`